src/App.css

```diff
@@ -1206,4 +1206,19 @@
     border-top: 1px solid var(--border-color);
     padding-top: 24px;
   }
+} 
+
+.activity-type-select {
+  width: 100%;
+  padding: 8px;
+  margin-top: 8px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  background-color: white;
+  font-size: 14px;
+}
+
+.activity-type-select:focus {
+  outline: none;
+  border-color: #007bff;
 } 
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
.activity-type-select {
  width: 100%;
  padding: 8px;
  margin-top: 8px;
  border: 1px solid var(--border-color);
  border-radius: 4px;
  background-color: var(--task-bg);
  color: var(--text-primary);
  font-size: 14px;
}

.activity-type-select:focus {
  outline: none;
  border-color: var(--primary-color);
}
```
Use CSS variables for consistency with the existing theme system instead of hardcoded colors. This maintains the dark/light theme compatibility and improves maintainability.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;